ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -75,29 +83,43 @@ dump_repos() {
 
 env > /root/last_run-env.txt
 if ! grep ":$MY_UID:" /etc/group; then
-  groupadd -g "$MY_UID" jenkins
+  if ! groupadd -g "$MY_UID" "${REMOTE_ACCT:-jenkins}"; then
+    echo "Couldn't add group ${REMOTE_ACCT:-jenkins} "\
+         "with gid $MY_UID, pressing on..."
+  fi
 fi
 mkdir -p /localhome
-if ! grep ":$MY_UID:$MY_UID:" /etc/passwd; then
-  useradd -b /localhome -g "$MY_UID" -u "$MY_UID" -s /bin/bash jenkins
+if grep ":$MY_UID:$MY_UID:" /etc/passwd; then
+  home=$(grep "$MY_UID:$MY_UID::" /etc/passwd | cut -d: -f 6)
+else
+  if ! useradd -b /localhome -g "$MY_UID" -u "$MY_UID"
+               -s /bin/bash jenkins; then
```

Comment:
(lint) This flag is used as a command name. Bad line break or missing [ .. ]? [SC2215]